packages/editor/src/components/post-publish-panel/test/__snapshots__/index.js.snap

```diff
@@ -96,7 +96,7 @@ exports[`PostPublishPanel should render the post-publish panel if the post is pu
                   post address
                 </label>
                 <input
-                  class="components-text-control__input"
+                  class="post-publish-panel__postpublish-post-address components-text-control__input"
```

Comment:
This snapshot having changed is also an indication that the classnames change above is introducing some side effects